bagit.py

```diff
@@ -1144,6 +1144,10 @@ def make_manifests(data_dir, processes, algorithms=DEFAULT_CHECKSUMS, encoding='
     byte_value_set = set(total_bytes.values())
     file_count_set = set(num_files.values())
 
+    # allow a bag with an empty payload
+    if not (len(byte_value_set) and len(file_count_set)):
```

Comment:
I think it would be more pythonic to use the shorter `if not byte_value_set and not file_count_set:` version

Reply:
No problem.  Will modify the statement and add a test case.